.github/workflows/sphinx-docs.yml

```
@@ -0,0 +1,60 @@
+            --sep docs/source
+        continue-on-error: true
+
+      - name: Build Sphinx documentation
+        run: |
+          cd backend
+          sphinx-build -b html docs/source docs/build || echo "Sphinx build completed with warnings"
+        continue-on-error: true
+        # Documents your FastAPI backend for agent context
+
+      - name: Deploy docs to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.8)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)